ui/main/src/assets/i18n/en.json

```diff
@@ -173,7 +173,9 @@
       "title": "Sort by",
       "severity": "Criticality then date",
       "publish": "Date",
-      "unread": "Unread then date"
+      "unread": "Unread then date",
+      "startDate": "Start date",
+      "endDate": "End date"
```

Comment:
Replace : 
 "Date" -->  "Date of reception" 
 "Unread then" --> "Unread then date of reception"
 "Criticality then" --> "Criticality then date of reception"